rust/pst/scripts/generate-ts.ts

```diff
@@ -0,0 +1,33 @@
+// NOTE: It is currently required to use a script file in order to run json2ts instead of using its
```

Comment:
shouldn't the implementation also have the "auto-generation" warn/notice in the header?
<img width="889" alt="image" src="https://user-images.githubusercontent.com/84311757/195581357-6a20162d-142c-4f79-af80-57079b417701.png">

Reply:
I don't like the 'any'. Shouldn't the `Input` generic type be set to the `Balance` (since it now seem to contain the 'function' field).

Or maybe even better - 
1. define a separate type - sth like
```
export interface BaseInput {
 function: string
}
```

2. remove the "function" from all the other types (like `Balance` - in order not to confuse the users with 'function' field)
3. define the generic like this `<BaseInput & Balance>`
<img width="663" alt="image" src="https://user-images.githubusercontent.com/84311757/195584831-97907196-f9cc-488b-9d79-4647492a1d8e.png">

Reply:
ok, summing-up!
1. unknown type has been fixed
2. evaluationOptions have been moved to a separate method
3. ReadAction and ReadResponse have been merged to one View file
4. optional types have been fixed
5. union types are generated automatically by the json-to-typescript lib as the json schema is an enum containing types, i don't think it's an issue of a great matter so i'll leave it as is
6. auto-generation comment has been added
7. no more any, sir